File: CST ASP.NET @ PCC for MDAC/ASP.NET Solution Folder/Allfiles/Democode/Mod06CS/CSharpClassLibrary/Class1.cs
using System;

namespace CSharpClassLibrary
{
	/// <summary>
	/// Summary description for Class1.
	/// </summary>
	public class Class1
	{
		public Class1()
		{
			//
			// TODO: Add constructor logic here
			//
			//HttpContext.Current.Trace.IsEnabled=false;
			//HttpContext.Current.Trace.Write("component", "constructor");
		}
		public string hello()
		{
			//HttpContext.Current.Trace.IsEnabled=false;
			//HttpContext.Current.Trace.Write("component", "hello");
			return ("Hi from C# component");
		}
	
		public Single ComputeShipping(Single fPrice)
		{
			Single fShipping;

			if (fPrice > 20) 
				fShipping = fPrice * 0.2F;
			else
				fShipping = fPrice * 0.1F;

			return(fShipping);
		}
	}

	
}

File: CST ASP.NET @ PCC for MDAC/ASP.NET Solution Folder/Allfiles/Democode/Mod06CS/VBClassLibrary/Class1.vb
Public Class Class1
    Function hello() As String
        Return "Hi from Visual Basic .NET component"
    End Function

    Function ComputeShipping(ByVal sngPrice As Single) As Single

        Dim sngShipping As Single

        If sngPrice > 15 Then
            sngShipping = sngPrice * 0.1
        Else
            sngShipping = sngPrice * 0.05
        End If

        Return (sngShipping)

    End Function
End Class

